src/web/components/ClientSideTokenGeneration/CstgDomainsTable.scss

```diff
@@ -43,7 +47,34 @@
 
     &-right {
       display: flex;
-      align-items: center;
+      justify-content: right;
+    }
+
+    .domains-search-bar-container {
+      display: flex;
+      border-bottom: 2px solid var(--theme-action);
+      padding: 6px 2px;
+      ::placeholder {
+        color: var(--theme-action);
+        opacity: 1;
+      }
+      margin-right: 20px;
+    }
+
+    .domains-search-bar-icon {
+      color: var(--theme-action);
+      height: 16px;
+      margin-left: 8px;
+    }
+
+    .domains-search-bar {
+      width: 100%;
+      border: none;
+      outline: none;
+      color: var(--theme-action);
+      font-weight: 400;
+      font-size: 0.75rem;
+      line-height: 0.9375rem;
```

Comment:
Can we replace this with a unitless line-height? i.e. no `rem`. Otherwise it can lead to issues where the font is too big for the line-height.

Reply:
I think we can probably just remove this line entirely